tests/labhub_test.py

```diff
@@ -193,20 +201,27 @@ def test_unassign_cmd(self):
         mock_iss.assignees = (None, )
         mock_iss.unassign = MagicMock()
 
-        testbot.assertCommand('!unassign https://github.com/coala/name/issues/23',
-                              'you are unassigned now', timeout=10000)
+        # Start ignoring InvalidLinkBear
```

Comment:
change all of these invalid URLs to contain constant strings which you can ignore using settings in the .coafile, and can be re-used in future so `InvalidLinkBear` doesnt need to be ignored all the time.

(and where you are improving the style of tests, also switch to using consistent issue numbers instead of random numbers, which make the reader wonder whether those numbers have some impact on the tests.)

https://github.com/coala/name/issues/23 -> https://github.com/coala/example/issues/999

https://github.com/coala/s/issues/52 -> https://github.com/coala/example2/issues/999

https://gitlab.com/ala/am/issues/532 -> https://gitlab.com/example/test/issues/999